android/app/build.gradle

```diff
@@ -85,5 +85,6 @@ flutter {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'com.facebook.android:facebook-login:latest.release'
```

Comment:
change latest.release to a concrete version plz (like 1.0.0 (but actual for fb))